client/components/platform-checkout/save-user/agreement.js

```diff
@@ -10,24 +10,24 @@ const Agreement = () => {
 		<div className="tos">
 			{ interpolateComponents( {
 				mixedString: __(
-					'By entering your phone number and completing your purchase, you will create a Platform Checkout account and agree to {{termsOfService/}} and {{privacyPolicy/}}.',
+					'By entering your phone number and completing your purchase, you will create a WooPay account and agree to {{termsOfService/}} and {{privacyPolicy/}}.',
```

Comment:
this should read:

By entering your mobile phone number and completing your purchase, you will create a WooPay account, and you agree to the Terms of Service and Privacy Policy

Reply:
Updated in https://github.com/Automattic/woocommerce-payments/pull/4107/commits/b3da826f56743af9f316ba7a8d5c8b05f2e95664